### ISSUE TITLE
JSON API page should mention new listing of Discover collections/tagmoji

### ISSUE THREAD

**Issue by sardisson** [microdotblog/help.micro.blog#28]
The [JSON API](https://help.micro.blog/2017/api-json/) help page should mention the update to `/posts/discover` to now include the list of tagmoji Discover collections that @manton [announced today](https://www.manton.org/2019/07/08/new-tagmoji-on.html):

> The /posts/discover API for Micro.blog apps has also been updated. It now returns the emoji that Micro.blog is currently tracking, and whether it’s one that is “featured” in the default popup menu in Micro.blog on the web.

**Comment by sardisson** [comment]
No, that’s the same thing that’s been there since 2017. The endpoint has been in the docs since then, but what it returns has changed (which is the part of the doc that needs updating). The endpoint now also returns a `tagmoji` dictionary in the `_microblog` section, in addition to the original “JSON for featured posts” `items` (posts) dictionary.

**Comment by simon-woods** [comment]
Oh! I see. Agree that the update should be referenced/explained.
